speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
The latter issue you mention is a known one.  It's the reason I want to add generic search/play functionality.  Currently, you cannot say "play ghostbusters."  It *can* work, but it can also just pick some other somewhat similar intent too.

It's a bug only in the sense that we don't outright fail on it at the moment.  I haven't bothered trying to catch it, because it will be fixed entirely after I merge in the generic play/search branch I have.

I haven't personally had any issues speaking articles even if they're not present in the sample utterances.  If you're having problems, that's a good enough reason to add them.  I only commented on it because we were just discussing trimming the utterances, not expanding them ;)

But I don't care one way or the other provided it all works.